compiler/llvm/llvmUtil.cpp

```diff
@@ -258,9 +258,9 @@ llvm::Value *convertValueToType(
       llvm::Value* tmp_alloc;
       if( layout.getTypeStoreSize(newType) >=
           layout.getTypeStoreSize(curType) )
-        tmp_alloc = createTempVarLLVM(irBuilder, newType, "");
+        tmp_alloc = createLLVMAlloca(irBuilder, newType, "");
       else {
-        tmp_alloc = createTempVarLLVM(irBuilder, curType, "");
+        tmp_alloc = createLLVMAlloca(irBuilder, curType, "");
```

Comment:
Please move convertValueToType into compiler/codegen/expr.cpp and make it call createVarLLVM (so that the lifetime start / end intrinsics are generated for these temporaries).

convertValueToType is not currently called in any of the LLVM code we want llvmUtil to keep independent from the rest of the Chapel compiler.